ipaserver/install/replication.py

```diff
@@ -232,6 +232,23 @@ def get_ds_version(conn):
     return vendor_version
 
 
+def get_replication_plugin_name(dirsrv_get_entry):
+    # Support renaming of a replication plugin in 389-ds
+    # IPA topology plugin depends on the replication plugin but
+    # 389-ds cannot handle older alias querying in the plugin
+    # configuration with 'nsslapd-plugin-depends-on-named: ..' attribute
+    #
+    # dirsrv_get_entry: function (dn, attrs) -> str
+    # returns entry dictionary
+    try:
+        entry = dirsrv_get_entry(
+            "cn=Multisupplier Replication Plugin,cn=plugins,cn=config",
+            ["cn"])
+        return entry['cn']
+    except Exception:
```

Comment:
can the Exception be more specific?

Reply:
Unfortunately, no. There are two different `get_entry` implementations (one from IPA, one from lib389) and they throw different exceptions.